src/main/kotlin/no/nav/helse/fritakagp/processing/brukernotifikasjon/BrukernotifikasjonService.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.helse.fritakagp.processing.brukernotifikasjon
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.hag.utils.bakgrunnsjobb.Bakgrunnsjobb
+import no.nav.helsearbeidsgiver.utils.log.logger
+import no.nav.tms.varsel.action.Sensitivitet
+import no.nav.tms.varsel.action.Tekst
+import no.nav.tms.varsel.action.Varseltype
+import no.nav.tms.varsel.builder.VarselActionBuilder
+import java.time.ZonedDateTime
+
+class BrukernotifikasjonService(
+    private val om: ObjectMapper,
+    private val sensitivitetNivaa: Sensitivitet = Sensitivitet.High,
+    private val frontendAppBaseUrl: String = "https://arbeidsgiver.nav.no/fritak-agp"
+) {
+    private val logger = this.logger()
+    fun opprettVarsel(varselId: String, jobb: Bakgrunnsjobb): String {
+        val jobbData = om.readValue<BrukernotifikasjonJobbdata>(jobb.data)
```

Comment:
Mulig jeg tar feil her. Men vil ikke gamle bakgrunnsjobber feile her siden jobbdata har endret seg?
Kanskje det er lurt å gjøre det i to steg. så man har ny og gammel processor samtidig. men bare lager nye jobber av ny type. deretter kan man slette gammel processor når alle er kjørt igjennom feks.